builtin/game/misc.lua

```diff
@@ -164,6 +164,12 @@ function core.record_protection_violation(pos, name)
 	end
 end
 
+-- To be overridden by Creative mods
+
+local creative_mode_cache = core.settings:get_bool("creative_mode", false)
+function core.creative_is_enabled_for(name)
```

Comment:
I propose renaming this to `creative_is_enabled` (or maybe `creative_enabled`). The naming with a verb and `_for()` is not used anywhere else.